src/aeboards/aegis/aeboards_aegis.json

```diff
@@ -2,148 +2,164 @@
   "name": "AEboards AEGIS",
   "vendorId": "0x4145",
   "productId": "0x0807",
-  "lights": 0,
+  "supports": {
+    "rotaryEncoders": true,
+    "lighting": {
+      "keycodes": "WT",
+      "colors": 2,
+      "brightness": true,
+      "speed": true,
+      "patterns": ["Off", "Let's Go", "Five"]
```

Comment:
There is currently only three "lighting" implementations, "none", "wt_rgb_backlight", "wt_mono_backlight".
If it's not "none", those implementations will handle the commands already defined, so the "keycodes", "colors", "brightness", "speed" values are redundant. These are constant for the implementation... if you want to generalize the code this way, that's fine, it just doesn't make sense to require the user to set them in the JSON.

"patterns" should be called "effects" since that is the name I use in the code and the keycodes.
It's a good idea and will allow someone to change which effects are built into a given firmware, they don't have to be the ones I use for my PCBs, and also deals with something like M6-B which could have different effects to M60-A etc.

If "wt_mono_backlight" true, there could be one parameter, "rgb_indicator" which can enable the color picker for the RGB LED on Xeno/WT75-A and future PCBs.

If "wt_rgb_backlight" there could be one parameter, "custom_colors" which can enable the 6 color pickers for M6-B.